game/client_types/logic.js

```diff
@@ -92,14 +97,23 @@ module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
     function finalizeRound(total, contribs, sortedContribs) {
         var i, contribObj;
         var pid, payoff, client, distribution;
+        
+        //console.log(contribs);
+        //console.log(total);
 
         distribution = total / sortedContribs.length;
         // Save the results for each player, and notify him/her.
         for ( i=0 ; i < sortedContribs.length; i++) {
             contribObj = sortedContribs[i];
             pid = contribObj.player;
 
-            payoff = distribution + contribs[pid] - contribObj.contribution;
+            payoff = node.game.settings.COINS - contribs[pid] + distribution; // Edited FrK: - contribObj.contribution;
```

Comment:
Not sure why you changed this, maybe we can chat about this.

Reply:
Because `contribs[pid]` and `contribObj.contribution` are of equal value, so the resulting `payoff` is just equal to `distribution`. So I take `node.game.settings.COINS - contribs[pid]` as saving to personal account.